src/nomad_simulations/schema_packages/model_system.py

```diff
@@ -358,55 +364,58 @@ class Cell(GeometricSpace):
-            raise NotImplementedError
+    # @staticmethod
+    # def _generate_comparer(obj: 'Cell') -> 'Generator[Any, None, None]':
+    #     try:
+    #         return ((HashedPositions(pos)) for pos in obj.positions)
+    #     except AttributeError:
+    #         raise NotImplementedError
 
-    @catch_not_implemented
-    def is_lt_cell(self, other) -> bool:
-        return set(self._generate_comparer(self)) < set(self._generate_comparer(other))
+    # @catch_not_implemented
+    # def is_lt_cell(self, other) -> bool:
+    #     return set(self._generate_comparer(self)) < set(self._generate_comparer(other))
 
-    @catch_not_implemented
-    def is_gt_cell(self, other) -> bool:
-        return set(self._generate_comparer(self)) > set(self._generate_comparer(other))
+    # @catch_not_implemented
+    # def is_gt_cell(self, other) -> bool:
+    #     return set(self._generate_comparer(self)) > set(self._generate_comparer(other))
 
-    @catch_not_implemented
-    def is_le_cell(self, other) -> bool:
-        return set(self._generate_comparer(self)) <= set(self._generate_comparer(other))
+    # @catch_not_implemented
+    # def is_le_cell(self, other) -> bool:
+    #     return set(self._generate_comparer(self)) <= set(self._generate_comparer(other))
 
-    @catch_not_implemented
-    def is_ge_cell(self, other) -> bool:
-        return set(self._generate_comparer(self)) >= set(self._generate_comparer(other))
+    # @catch_not_implemented
+    # def is_ge_cell(self, other) -> bool:
+    #     return set(self._generate_comparer(self)) >= set(self._generate_comparer(other))
 
-    @catch_not_implemented
-    def is_equal_cell(self, other) -> bool:  # TODO: improve naming
-        return set(self._generate_comparer(self)) == set(self._generate_comparer(other))
+    # @catch_not_implemented
+    # def is_equal_cell(self, other) -> bool:  # TODO: improve naming
+    #     return set(self._generate_comparer(self)) == set(self._generate_comparer(other))
 
-    def is_ne_cell(self, other) -> bool:
-        # this does not hold in general, but here we use finite sets
-        return not self.is_equal_cell(other)
+    # def is_ne_cell(self, other) -> bool:
+    #     # this does not hold in general, but here we use finite sets
+    #     return not self.is_equal_cell(other)
 
 
 class AtomicCell(Cell):
     """
     A base section used to specify the atomic cell information of a system.
     """
 
-    atoms_state = SubSection(sub_section=AtomsState.m_def, repeats=True)
+    # this can be found under ModelSystem
+    # atoms_state = SubSection(sub_section=AtomsState.m_def, repeats=True)
 
-    n_atoms = Quantity(
-        type=np.int32,
-        description="""
-        Number of atoms in the atomic cell.
-        """,
-    )
+    # this is moved to ModelSystem.
+    # n_atoms = Quantity(
+    #     type=np.int32,
+    #     description="""
+    #     Number of atoms in the atomic cell.
+    #     """,
+    # )
 
     equivalent_atoms = Quantity(
```

Comment:
Do you know in what context this is used? I presume that it has something to do with calculating some minimal cell representation?

Reply:
Just add a "#?" comment that asks the use of this